### ISSUE TITLE
Problem with Rbminer's binary

### ISSUE THREAD

**Issue by neraserpent** [josepcarmona/PMLAB#7]
Rbminer (downloaded from the [site](http://people.ac.upc.edu/msole/homepage/rbminer.html) listed in pm_guide) doesn't respond to the examples proposed in it's manual. 
All the binaries from the archive seem legit and working except for the rbminer bin itself - as shown in the screens below.
![2017-06-18_04-45-20](https://user-images.githubusercontent.com/14029885/27257486-33375612-53e1-11e7-9213-fddc4e92058a.png)
![2017-06-18_05-35-41](https://user-images.githubusercontent.com/14029885/27257682-05dee4c6-53e8-11e7-8790-e012984b6332.png)
![2017-06-18_04-46-24](https://user-images.githubusercontent.com/14029885/27257488-49348232-53e1-11e7-9006-42fabd0ecc0e.png)
Are there any dependencies or prerequisites for the rbminer itself? Is the link to Rbminer application in pmlab/doc/pm_guide.pdf correct (the link to stp isn't)?
Is there any way to get a proper rbminer binary if the one from the site is really broken?

I'm sorry if it's inappropriate to ask questions about a separate project here, but i couldn't find a better place to do it.

**Comment by josepcarmona** [comment]
It looks like you have not installed the rbminer tool properly. Please follow the manual so that the tool is available in the PATH variable.

**Comment by neraserpent** [comment]
I truly hope to be the only source of the problem, yet every workaround with the PATH has given the same result. 
![2017-06-21_19-36-46](https://user-images.githubusercontent.com/14029885/27395820-c44d2764-56b9-11e7-9384-c95f03729ef3.png)
One of the possible reasons for this might be the fact that i run your code from Docker - it probably shouldn't be an issue, but i'll also try a normal system (just to be sure) and text you back right after.

**Comment by neraserpent** [comment]
The problem was solved by loading rbminer's unspecified dependencies (found them in ldd) and tools for running 32-bit apps. 
After that the bloody mayhem of graph_tools' dependencies has begun. I've battled it for 2 weeks and still didn't get to the end. 
It's quite a pity to turn away from the Process Mining methodology due to the hardships of deployment of the only tool that can be integrated into production, but that seems inevitable. I'll try to save some functionality by replacing/rewriting Graph objects with something primitive, though it may get things even worse.

With best regards and a hope that one day the project will be a bit more noob-friendly
